src/server/server.ts

```diff
@@ -104,6 +114,7 @@ registerRoute('get', '/run/:host/:id', (req, res) => {
                 returnUrl: '',
                 origin: currentConfig.editorUrl,
                 host: host,
+                assets: assets,
```

Comment:
Though per one of the comments above, you could just make it a global and not need to evaluate the function at all.

---
In reply to: [126536827](https://github.com/OfficeDev/script-lab/pull/474#discussion_r126536827) [](ancestors = 126536827)